doc/man3/CMS_EnvelopedData_create.pod

```diff
@@ -0,0 +1,48 @@
+=pod
+
+=head1 NAME
+
+CMS_EnvelopedData_create - Create CMS envelope
+
+=head1 SYNOPSIS
+
+ #include <openssl/cms.h>
+
+ CMS_ContentInfo *CMS_EnvelopedData_create(const EVP_CIPHER *cipher);
+
+=head1 DESCRIPTION
+
+The B<CMS_ContentInfo>  
```

Comment:
This seems to be a half finished sentence

Reply:
Ah that should not be there. Not sure how it got there. Fixed.